leapp/libraries/stdlib/call.py

```diff
@@ -102,6 +104,12 @@ def _call(command, callback_raw=lambda fd, value: None, callback_linebuffered=la
         raise ValueError('poll_timeout parameter has to be integer greater than zero')
     if not isinstance(read_buffer_size, int) or isinstance(read_buffer_size, bool) or read_buffer_size <= 0:
         raise ValueError('read_buffer_size parameter has to be integer greater than zero')
+    if find_executable(command[0]) is None:
```

Comment:
Why are we doing this? I would like to veto this part - the error you are raising is not because the command actually failed but because it does not exist 
This is a different kind of error and if you are not happy with handling OSErrors like subprocess and all other to me known apis raise then replace it something that is distinguishable.

Besides if the path exist and it is not executable IIRC you would actually get a different kind of error.
You are trying to reinvent the wheel here for something that already is properly handled on a lower layer of this code. So rather handle and transform the exception than trying make your own checks that properly not even cover everything